package.json

```diff
@@ -83,5 +78,8 @@
     "react-redux": "5.0.7",
     "redux": "4.0.0",
     "redux-saga": "0.16.0"
+  },
+  "resolutions": {
+    "babel-core": "7.0.0-bridge.0"
```

Comment:
This "resolution" field is yarn-specific, right? Is there a conflict that requires to force this `babel-core` release?

Reply:
It is yarn specific. One dependency (`babel-jest` AFAIK) loads babel 6 in our specific circumstances and therefore requires us to use this resolution + bridge to force babel 7.

I would have put in a comment but unfortunately JSON can't handle comments :(

Reply:
I'll add something in a commit so the info can at least be found through a `git blame`